src/universal/redux/__tests__/AuthEngine.test.js

```diff
@@ -31,7 +31,6 @@ test('saveToken saves token', (done) => {
   const cb = (err, token) => {
     expect(err).toBe(null);
     expect(token).toBe(testToken);
-    expect(segmentActions.segmentEventIdentify).toBeCalled();
```

Comment:
Yet there still is a case where we _do_ want the `segmentEventIdentify` method to be called, so let's test it!

Reply:
we test it, and we call it,  just not on auth token renewals, because that won't give us anything new.